dcos/marathon.py

```diff
@@ -388,6 +388,35 @@ def scale_app(self, app_id, instances, force=None):
         deployment = response.json()['deploymentId']
         return deployment
 
+    def scale_group(self, group_id, scale_factor, force=None):
+        """Scales a group with the requested scale-factor.
+        :param group_id: the ID of the group to scale
+        :type group_id: str
+        :param scale_factor: the requested value of scale-factor
+        :type scale_factor: int
+        :param force: whether to override running deployments
+        :type force: bool
+        :returns: the resulting deployment ID
+        :rtype: bool
+        """
+
+        group_id = self.normalize_app_id(group_id)
+
+        if not force:
+            params = None
+        else:
+            params = {'force': 'true'}
+
+        url = self._create_url('v2/groups{}'.format(group_id))
+
+        response = http.put(url,
+                            params=params,
+                            json={'scaleBy': float(scale_factor)},
```

Comment:
Let's handle the error much better. I would pydoc for this method to expect a `float` and parse the input in the command: `cli/dcoscli/marathon/main.py`. We have code that parses `strings` at `int` and throws a `DCOSException` instead of a `ValueError`: https://github.com/mesosphere/dcos-cli/blob/master/dcos/util.py#L438-L454.

Let's do the same thing for `float` and add a function to the module `dcos.util` that parses as a float and throws a `DCOSException` if it fails.